example/otel-collector/Makefile

```diff
@@ -3,15 +3,15 @@ namespace-k8s:
 
 jaeger-operator-k8s:
 	# Create the jaeger operator and necessary artifacts in ns observability
-	kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd.yaml
-	kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/service_account.yaml
-	kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role.yaml
-	kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role_binding.yaml
-	kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/operator.yaml
+	kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.31.0/deploy/crds/jaegertracing.io_jaegers_crd.yaml
```

Comment:
Try https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.31.0/deploy/crds/jaegertracing.io_jaegers_crd.yaml 😄 

Check https://github.com/jaegertracing/documentation/pull/542/files

Reply:
sorry about that, I've updated the version info.

Reply:
Any reason why not using https://github.com/jaegertracing/jaeger-operator/releases/download/v1.31.0/jaeger-operator.yaml as documented?

Reply:
@pellared please take another look.